.travis.yml

```diff
@@ -8,7 +8,7 @@ services:
  - docker
 
 go:
- - 1.9
+ - 1.10.x
```

Comment:
let's build on both 1.10 and 1.11 until we have a reason to not use 10 (eg, modules)